docs/sources/panels-visualizations/visualizations/time-series/index.md

```diff
@@ -56,7 +56,26 @@ refs:
     - pattern: /docs/grafana/
       destination: /docs/grafana/<GRAFANA_VERSION>/panels-visualizations/configure-standard-options/#max
     - pattern: /docs/grafana-cloud/
+<<<<<<< HEAD
       destination: /docs/grafana-cloud/visualizations/panels-visualizations/configure-standard-options/#max
+=======
+      destination: /docs/grafana-cloud/visualizations/panels-visualizations/configure-overrides/#add-a-field-override
+  configure-field-overrides:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/panels-visualizations/configure-overrides/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/visualizations/panels-visualizations/configure-overrides/
+  alert-list:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/panels-visualizations/visualizations/alert-list/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/visualizations/panels-visualizations/visualizations/alert-list/
+  link-alert:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/alerting-rules/create-grafana-managed-rule/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/alerting-and-irm/alerting/alerting-rules/create-grafana-managed-rule/
+>>>>>>> 000b5dd6eab (docs: alerting: clarify which panels support alerts (#88756))
```

Comment:
Git conflict markers are also here

Reply:
oh i was actually wondering if i should delete the markers!
so that's all i have to do right ? @jdbaldry

Reply:
Yeah, you need to delete the markers and modify the file so that its contents are what you want.

We have some documentation (which links to more documentation) in https://grafana.com/docs/writers-toolkit/write/tooling-and-workflows/#resolve-conflicts